build.gradle

```diff
@@ -26,6 +26,9 @@ allprojects {
   ext.gradleScriptDir = "${rootProject.projectDir}/gradle"
   //archivesBaseName = 'archie'
 
+  tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:all', '-quiet')
+  }
```

Comment:
Is this still needed?

Reply:
Yes. On semaphore this defaults to -Xdoclint:none. The quiet should not be needed, but without it this does not work, so I added it again.